python/lsst/summit/utils/tmaUtils.py

```diff
@@ -150,11 +165,88 @@ def getAzimuthElevationDataForEvent(client, event, prePadding=0, postPadding=0):
                                prePadding=prePadding,
                                postPadding=postPadding)
 
+    azValues = azimuthData['actualPosition'].values
+    elValues = elevationData['actualPosition'].values
+    azDemand = azimuthData['demandPosition'].values
+    elDemand = elevationData['demandPosition'].values
+
+    azError = (azValues - azDemand) * 3600
+    elError = (elValues - elDemand) * 3600
+
+    azimuthData['azError'] = azError
+    elevationData['elError'] = elError
+
     return azimuthData, elevationData
 
 
-def plotEvent(client, event, fig=None, prePadding=0, postPadding=0, commands={},
-              azimuthData=None, elevationData=None):
+def filterBadValues(values, maxDelta=0.1):
+    """Filter out bad values from a dataset, replacing them in-place.
+
+    This function replaces non-physical points in the dataset with an
+    extrapolation of the preceding two values. No more than 3 successive data
+    points are allowed to be replaced. Minimum length of the input is 3 points.
```

Comment:
I think anyone reading this in the future will wonder why do this at all instead of filtering/masking or highlighting the bad points in a plot? And I expect the answer will be some form of "we have a good reason to do it this way", but if you already have one or more in mind perhaps it's worth noting here.